docs/SNMP_Alarms.md

```diff
@@ -41,3 +41,13 @@ obtain the brief description, detailed description, and severity for the alarm:
 ## Alarm Models
 
 The alarm models used by Clearwater are defined in [alarmdefinition.h](https://github.com/Metaswitch/cpp-common/blob/master/include/alarmdefinition.h).
+
+## Alarm Resiliency
+
+Alarms are reported in such a way that if a component receiving alarm
```

Comment:
FTR I don't think line breaks matter - the first paragraph in this file is broken into separate lines with no spaces at the end of lines, but it's joined into one paragraph on ReadTheDocs and there are spaces between them (view-source:http://clearwater.readthedocs.org/en/latest/SNMP_Alarms.html)